applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -470,13 +491,62 @@ static void le_audio_msg_sub_thread(void)
 
 		default:
 			LOG_WRN("Unexpected/unhandled le_audio event: %d", event);
+
 			break;
 		}
 
 		STACK_USAGE_PRINT("le_audio_msg_thread", &le_audio_msg_sub_thread_data);
 	}
 }
 
+/* Handle content control events */
+static void content_control_msg_sub_thread(void)
+{
+	int ret;
+
+	const struct zbus_channel *chan;
+
+	while (1) {
```

Comment:
Not sure what the default is in NCS, but at least in Zephyr we use boolean values for boolean expression, e.g. this would be 
```suggestion
	while (true) {
```

Reply:
Searching the nrf-folder in NCS I got 74 matches for while (1) and 91 matches for while (true) :P  In our application files, however, I got 2 matches for while (true) and 7 for while (1)

Reply:
I'll therefore keep it as while (1) and change the remaining two to match, so we at least are consistent in our application. Easy enough to change later on